Automattic/woocommerce-payments
# Simplify deposits UI - Don't show deposits history for stores that have 0 recent deposits

Fixes #6001

#### Changes proposed in this Pull Request









This PR changes the look of the deposits overview component for new WC Pay merchants that have no deposits yet and are still in their 7-day waiting period.

Changes made:
 - Move the deposits history section heading into the `recent-deposits-list.tsx` since that's where we fetch the recent deposits
 - Send the account prop to the `RecentDepositsList` component as it's needed for the header

| Old UI  | New UI |
|:-------------: |:-------------: |
| ![image](https://user-images.githubusercontent.com/2275145/231663015-6cf53a08-f973-4f7a-90e4-45b5d0276f25.png)| ![image](https://user-images.githubusercontent.com/2275145/231663133-2c5a3398-f244-460a-859c-79c7ac1ac2b3.png)|

Note: the padding beneath the notice will be fixed by #6002

##### Why this change?
- It doesn't make sense to display the deposits history section (including deposits schedule section) when the merchant is still in their waiting period.
- Avoids showing conflicting information (deposits held for 7 days vs deposits dispatched every day)
- Matches the latest designs (see below)

![image](https://user-images.githubusercontent.com/2275145/231663629-7021c904-60d5-4b2d-8c33-27ef37b81475.png)


#### Testing instructions





1. Turn on the new deposit UI: `update_option( '_wcpay_feature_simplify_deposits_ui', 1 );`
2. Onboard as a new merchant or switch to a currency that has no pending/paid deposits yet.
3. Visit the Overview > Deposits section
4. On `develop` you'll see a deposits history section
5. On this branch, the deposits history will only show if you have deposits.

-------------------


**Post merge**
